src/js/jsx/tools/SamplerOverlay.jsx

```diff
@@ -371,6 +373,7 @@ define(function (require, exports, module) {
                             // Apply the color to selected layers
                             fluxActions.sampler.applyColor(this.state.document, null, sample.value);
                             d3.event.stopPropagation();
+                            headlights.logEvent("tools", "sampler-hud", sample.type);
```

Comment:
`tools > sampler-hud > fillColor` should be `tools > sampler-hud > fill-color`

you can use `_.kebabCase` to do the conversion.